src/actions/receiveAlert.js

```diff
@@ -0,0 +1,6 @@
+import { actionTypes } from '../constants';
+
+export default (message => ({
```

Comment:
it is unclear if this is a string or an object.  maybe the function can have (type, message, isDismissable, link, etc.) or at least provide a typehint for what the object shape should be.

Reply:
also, can the action creator create the id itself?